src/nix/hive/eval.nix

```diff
@@ -405,6 +403,8 @@ let
         }) config.deployment.keys;
     };
 
+    npkgs = hive.${name}.deployment.nixpkgs;
```

Comment:
`hive.${name}` is an unevaluated NixOS module, not a plain attribute set. Your [test](https://gist.github.com/cole-h/90054ea4b26b5b4bea03b5a979dbaaf1) accidentally works because `scadrial` happens to be just a set.